src/test/scala/org/globalforestwatch/summarystats/afi/AFiAnalysisSpec.scala

```diff
@@ -0,0 +1,74 @@
+package org.globalforestwatch.summarystats.afi
+
+import cats.data.{NonEmptyList, Validated}
+import com.github.mrpowers.spark.daria.sql.transformations.sortColumns
+import com.github.mrpowers.spark.fast.tests.DataFrameComparer
+import geotrellis.vector._
+import org.apache.sedona.core.spatialRDD.SpatialRDD
```

Comment:
Very minor comment - but looks like this line is an unneeded import, and also SparkSession can be removed from the import at line 11, and JobError from the import at line 15